contrib/grinder/README.md

```diff
@@ -1,99 +1,5 @@
 # Blueflood Grinder Integration
-##Intro
-Grinder is a distributed load testing tool described [here](http://grinder.sourceforge.net/g3/getting-started.html)
 
-This code defines implementations of grinder worker threads meant to repeatedly invoke the required number of BF api calls during each "reporting interval"
+### DEPRECATED
```

Comment:
We don't need the DEPRECATED comments--delete it all.  We should have some docs which point users to the perf repos, though.

Perhaps in the main README we have a section entitled "Related Repos" which can list all the things we use in conjunction with blueflood.  This would include (I'm sure there are others)
- perf tests repos
- carbon forwarder
- blueflood finder
- statsD plugin
- I'm sure there are more.....

We could put this in the wiki on github, but I wonder if its more noticeable in the README.